revscoring/scorers/tests/test_svc.py

```diff
@@ -1,34 +1,56 @@
+import random
 from io import BytesIO
+from itertools import chain
 
 from nose.tools import eq_
 
-from ...features import (proportion_of_badwords_added,
-                         proportion_of_symbolic_added)
+from ...features import Feature
+from ...languages import Language
 from ..scorer import MLScorerModel
-from ..svc import LinearSVC, RBFSVC, SVC
+from ..svc import LinearSVCModel, RBFSVCModel, SVCModel
 
 
+def process_float(): return float()
+some_float = Feature("some_float", process_float(),
+                      depends_on=[], returns=float)
+
+def process_other_float(): return float()
+other_float = Feature("other_float", process_other_float(),
+                      depends_on=[], returns=float)
+FEATURES = [some_float, other_float]
+
+
+def is_badword(word): return word == "badword"
+def is_misspelled(word): return word == "misspelled"
+some_language = Language(is_badword, is_misspelled)
+LANGUAGE = some_language
+
 def train_score(model):
-    train_stats = model.train([
-        ([10.1, 20.1], True),
-        ([19.2, 15.3], True),
-        ([13.1, 12.5], True),
-        ([0.5, 0.6], False),
-        ([2.4, 0.1], False),
-        ([0.9, 3.1], False)
-    ])
+    deterministic = random.Random(0)
+    observations = list(chain(
+        zip(((some, other) for some, other in
+             zip((deterministic.normalvariate(1, .3) for i in range(500)),
+                 (deterministic.normalvariate(2, .5) for i in range(500)))),
+            (True for i in range(100))),
+        zip(((some, other) for some, other in
+             zip((deterministic.normalvariate(-1, .5) for i in range(35)),
+                 (deterministic.normalvariate(-2, .3) for i in range(35)))),
+            (False for i in range(100)))
+    ))
```

Comment:
This syntax confuses me a little, so ignore me in case the question is stupid: aren't these `range(100)` supposed to be the same size as those `range(500)` and `range(35)`?